dunamai/__init__.py

```diff
@@ -281,7 +301,11 @@ def serialize(
             style = Style.Pep440
         out = ""
 
-        meta_parts = []
+        # treat local segments as the first meta_part
```

Comment:
A few lines up, the `if format is not None:` section should be updated to support a corresponding `{tagged_metadata}` placeholder, which will make it supported by the `--format` option as well.